src/main/java/stockservice/BasicStockService.java

```diff
@@ -0,0 +1,51 @@
+package stockservice;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+/**
+ * A concrete class that implements the StockService interface
+ * 
+ * @author Christopher Stafford
+ */
+public class BasicStockService implements StockService {
+    
+    /**
+     * Method to get StockQuote from service
+     * 
+     * @param symbol The symobl of the stock to pull
+     * @return The full stock quote as <CODE> StockQuote </CODE>
+     */
+    public StockQuote getQuote(String symbol) {
+        StockQuote quote = new StockQuote(Calendar.getInstance(), new BigDecimal("342.1"), "APPL");
+        
```

Comment:
since this a dummy (hard coded implementation) of the StockService there is no need to ever throw 
            throw new MissingResourceException("Stock symbol not found", "BasicStockService", symbol);

Reply:
I wrote this in the event that we're consuming an outside service. I know it's hard coded but I was trying to think ahead.